tests/kvasir-tests/Makefile.common

```diff
@@ -204,10 +204,12 @@ CFLAGS += -std=gnu89
 endif
 
 ifneq ($(OS_ID),CentOS)
+ifneq ($(OS_ID),Rocky Linux)
 # Ubuntu (17.10) sets -pie on by default
 # we must turn it back off
 CFLAGS += -no-pie
 endif
```

Comment:
What was the decision here?  Just do -no-pie always?

Reply:
That is a separate issue, which can be addressed in a different pull request.